conanfile.py

```diff
@@ -47,33 +46,29 @@ def config_options(self):
             del self.options.fPIC
 
     def build_requirements(self):
-        self.build_requires('protoc_installer/3.9.1@bincrafters/stable#0')
-        self.build_requires('grpc_codegen/1.27.3@{}'.format(self._orbit_channel))
+        self.build_requires('protobuf/3.21.4')
+        self.build_requires('grpc/1.48.0')
         self.build_requires('gtest/1.11.0', force_host_context=True)
 
     def requirements(self):
-        self.requires("abseil/20211102.0")
-        self.requires("bzip2/1.0.8", override=True)
+        self.requires("abseil/20220623.0")
         self.requires("capstone/4.0.2")
-        self.requires("grpc/1.27.3@{}".format(self._orbit_channel))
-        self.requires("c-ares/1.15.0", override=True)
-        self.requires("llvm-core/12.0.0@{}".format(self._orbit_channel))
-        self.requires("openssl/1.1.1k", override=True)
-        self.requires("outcome/2.2.0")
+        self.requires("grpc/1.48.0")
+        self.requires("llvm-core/12.0.0@orbitdeps/stable")
```

Comment:
That's the only custom conan package that remains. I will tackle that in a separate PR.